website/docs/docs/building-a-dbt-project/hooks-operations.md

```diff
@@ -37,10 +37,10 @@ Hooks are snippets of SQL that are executed at different times:
 
 Hooks are a more-advanced capability that enable you to run custom SQL, and leverage database-specific actions, beyond what dbt makes available out-of-the-box with standard materializations and configurations.
 
-<VersionBlock firstVersion="1.2">
-
 <Snippet src="hooks-to-grants" />
 
+<VersionBlock firstVersion="1.2">
+
```

Comment:
don't we want to swap this?
```suggestion
<VersionBlock firstVersion="1.2">

<Snippet src="hooks-to-grants" />

```

Reply:
This `<VersionBlock>` is for the text below, which talks about how you _don't_ want to use hooks for grants, and provides a few alternative examples of what you might want to use hooks for. The `<VersionBlock>` for the `hooks-to-grants` snippet is contained within the snippet (your idea, and a good one!)